src/tools/illink/src/ILLink.Shared/TypeSystemProxy/WellKnownType.cs

```diff
@@ -39,6 +40,7 @@ public static (string Namespace, string Name) GetNamespaceAndName (this WellKnow
 				WellKnownType.System_Runtime_CompilerServices_DisablePrivateReflectionAttribute => ("System.Runtime.CompilerServices", "DisablePrivateReflectionAttribute"),
 				WellKnownType.System_Void => ("System", "Void"),
 			};
+			#pragma warning restore CS8524 // The switch expression does not handle some values of its input type
```

Comment:
```suggestion
				WellKnownType.System_Void => ("System", "Void"),
				_ => default
			};
```
to avoid the warning suppression.